build/utils-node.js

```diff
@@ -495,7 +495,12 @@ module.exports = {
   },
 
   fileExists: function(path) {
-    return fs.existsSync(path);
+    try {
+      fs.accessSync(path);
+      return true;
+    } catch(err) {
+      return false;
+    }
```

Comment:
According to node.js API document, ```fs.exists``` / ```fs.existsSync``` are considered to be deprecated.

https://nodejs.org/dist/latest-v4.x/docs/api/fs.html#fs_fs_exists_path_callback